app/Models/Users.php

```diff
@@ -11,9 +13,20 @@ class Users extends Authenticatable
 
     protected $table = 'users';
     protected $primaryKey = 'user_ID';
-    protected $fillable = ['name', 'email', 'password'];
+    protected $fillable = ['name', 'email', 'password','description','avatar'];
     public $timestamps = false;
 
-    
+    public function library(): HasMany
+    {
+        return $this->hasMany(Library::class,"user_ID","user_ID");
+    }
+    public function booksInLibrary()
```

Comment:
advice: stay consistent. If you defined return types to other methods, define it for all